test/Microsoft.AspNet.Mvc.Core.Test/Formatters/XmlSerializerOutputFormatterTests.cs

```diff
@@ -8,6 +8,7 @@
 using System.Threading.Tasks;
 using Microsoft.AspNet.Http;
 using Microsoft.AspNet.Mvc.HeaderValueAbstractions;
+using Microsoft.AspNet.Mvc.Xml;
```

Comment:
Should we move these tests to a separate project..like `Micrsoft.AspNet.Mvc.Xml.Test`?